File: SEIST/polyglot/erlang/seis_release_contract.erl
-module(seis_release_contract).
-export([package/0, locales/0, live_upload_status/1]).

package() ->
  "dist/seis-static.zip".

locales() ->
  ["tr", "en", "fr", "it", "de", "es", "ar"].

live_upload_status(undefined) ->
  blocked_until_target_confirmed;
live_upload_status("") ->
  blocked_until_target_confirmed;
live_upload_status(_) ->
  ready_for_configured_target.

File: SEIST/polyglot/elixir/seis_release_contract.ex
defmodule SeisReleaseContract do
  @package "dist/seis-static.zip"
  @locales ["tr", "en", "fr", "it", "de", "es", "ar"]

  def package, do: @package
  def locales, do: @locales
  def supports_locale?(locale), do: locale in @locales
  def live_upload_status(nil), do: :blocked_until_target_confirmed
  def live_upload_status(_target), do: :ready_for_configured_target
end
